src/lib/Tab/Event/TabViewRenderEvent.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformAdminUi\Tab\Event;
+
+use Symfony\Component\EventDispatcher\Event;
+
+class TabViewRenderEvent extends Event
+{
+    /** @var string */
+    private $tabIdentifier;
+
+    /** @var string */
+    private $template;
+
+    /** @var array */
+    private $parameters;
+
+    /**
+     * @param string $tabIdentifier
+     * @param string $template
+     * @param array $parameters
+     */
+    public function __construct(string $tabIdentifier, string $template, array $parameters = [])
+    {
+        $this->tabIdentifier = $tabIdentifier;
+        $this->template = $template;
+        $this->parameters = $parameters;
+    }
+
+    /**
+     * @return string
+     */
+    public function getTabIdentifier(): string
+    {
+        return $this->tabIdentifier;
+    }
+
+    /**
+     * @param string $tabIdentifier
+     */
+    public function setTabIdentifier(string $tabIdentifier): void
```

Comment:
Should we change a tab identifier after creation?

Reply:
Good catch, no we should not.